### PR TITLE
raft: remove go vet compliants

### PR DESCRIPTION
**Pull request by mitake**
The values should be string for printf format.

### CODE REVIEW

File: raft/raft_paper_test.go [hunk 1]
```diff
@@ -460,7 +460,7 @@ func TestLeaderCommitEntry(t *testing.T) {
 			t.Errorf("to = %x, want %x", m.To, w)
 		}
 		if m.Type != pb.MsgApp {
-			t.Errorf("type = %s, want %s", m.Type, pb.MsgApp)
+			t.Errorf("type = %s, want %s", m.Type.String(), pb.MsgApp)
```
Comment (xiang90) [inline raft/raft_paper_test.go]: how about changing them to %v?

Comment (mitake) [inline raft/raft_paper_test.go]: ok, I'll fix them

**Comment by mitake** [comment]
@xiang90 updated, PTAL

**Comment by xiang90** [comment]
LGTM

**Pull request merged**